Find a functionally equivalent code:android.graphics.Canvas.drawRoundRect:COMMENT
Method Modifier: public      
Comment:/**
 * Draw the specified round-rect using the specified paint. The roundrect
 * will be filled or framed based on the Style in the paint.
 *
 * @param rect  The rectangular bounds of the roundRect to be drawn
 * @param rx    The x-radius of the oval used to round the corners
 * @param ry    The y-radius of the oval used to round the corners
 * @param paint The paint used to draw the roundRect
 */

@@ -1,6 +1,3 @@
 {
-    if (rect == null) {
-        throw new NullPointerException();
-    }
-    native_drawRoundRect(mNativeCanvas, rect, rx, ry, paint.mNativePaint);
+    drawRoundRect(rect.left, rect.top, rect.right, rect.bottom, rx, ry, paint);
 }

